### PR TITLE
Improve readme and package dependency

### PR DESCRIPTION
**Pull request by hubjac1**

### CODE REVIEW

File: packages/ozone-material/ozone-video-player/README.md [hunk 1]
```diff
@@ -64,6 +68,20 @@ Attribute are javaScript properties accessible from the dom.
 > video: Video
 > OzoneVideo to play.
 
+* markers
+>    markers: Array<MarkerOnVideo> 
+>    Array of markers on video
+
+* subtitlesAvailable
+> subtitlesAvailable: Array<string>
```
Comment (lmancel) [inline packages/ozone-material/ozone-video-player/README.md]: To indent or not ?

File: packages/ozone-material/ozone-item-edit/Readme.md [hunk 2]
```diff
@@ -24,13 +24,55 @@ ozone-localized-string: display ozone localized string
...
+<ozone-item-edit item-data={{item}} invalid="{{invalid}}">  </ozone-item-edit>
+```
+### attributes
+
+**itemData**: GenericItem item to display.
+**invalid**: boolean true if the value is invalid.
+
+### methods
+
+**getUpdatedData**: get the item with it's modifies fields.
```
Comment (lmancel) [inline packages/ozone-material/ozone-item-edit/Readme.md]:
```suggestion
**getUpdatedData**: get the item with its modified fields.
```

File: packages/ozone-material/ozone-item-edit/Readme.md [hunk 3]
```diff
@@ -24,13 +24,55 @@ ozone-localized-string: display ozone localized string
...
+
+### Events:
+
+**value-changed**: Trigger when a value has changed
```
Comment (lmancel) [inline packages/ozone-material/ozone-item-edit/Readme.md]:
```suggestion
**value-changed**: Triggered when a value has changed
```

File: packages/ozone-material/ozone-item-edit/Readme.md [hunk 4]
```diff
@@ -24,13 +24,55 @@ ozone-localized-string: display ozone localized string
...
+ 
+ 
+## ozone-edit-xxx
+
+### attributes
+
+**type**: string ozone type of the entry
+**value**: string value of the field
+**name**: LocalizedString name of the field
+**language**: string anguage to use in LocalizedName
```
Comment (lmancel) [inline packages/ozone-material/ozone-item-edit/Readme.md]:
```suggestion
**language**: string language to use in LocalizedName
```

File: packages/ozone-material/ozone-item-edit/Readme.md [hunk 5]
```diff
@@ -24,13 +24,55 @@ ozone-localized-string: display ozone localized string
...
+**disabled**: boolean Set to true to disable this input.
+**isModify**: boolean if the value is modify, is value will change to true.
+**invalid**: boolean true if the value is invalid
+**inputElement**: Returns a reference to the input element.
+
+### events
+
+ **value-changed**: Trigger when a value has changed
```
Comment (lmancel) [inline packages/ozone-material/ozone-item-edit/Readme.md]:
```suggestion
 **value-changed**: Triggered when a value has changed
```

File: packages/ozone-material/ozone-item-edit/Readme.md [hunk 6]
```diff
@@ -24,13 +24,55 @@ ozone-localized-string: display ozone localized string
...
+ **invalid-changed**: Trigger when invalid attribute has changed
```
Comment (lmancel) [inline packages/ozone-material/ozone-item-edit/Readme.md]:
```suggestion
 **invalid-changed**: Triggered when invalid attribute has changed
```

File: packages/ozone-typescript-client/Readme.md [hunk 7]
```diff
@@ -2,111 +2,150 @@
...
+
+	/**
+	 *  file file type client
+	 */
+	fileTypeClient(): FileTypeClient
+
+	/*
+        Insert the current Ozone session ID in the given URL ("/dsid=...).
+        This call
+        Throws an error if there is no session available.
```
Comment (lmancel) [inline packages/ozone-typescript-client/Readme.md]:
```suggestion
```

File: packages/ozone-typescript-client/Readme.md [hunk 8]
```diff
@@ -2,111 +2,150 @@
...
+	importExportClient(): ImportExportClient
+
+	/**
+	 *  file file type client
+	 */
+	fileTypeClient(): FileTypeClient
+
+	/*
+        Insert the current Ozone session ID in the given URL ("/dsid=...).
+        This call
```
Comment (lmancel) [inline packages/ozone-typescript-client/Readme.md]:
```suggestion
        This call throws an error if there is no session available.
```

File: packages/ozone-helper/ozone-media-url/src/ozone-media-url.ts [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+		this.id = id
+	}
+
+	/**
+	 * Convert uuid to ozone v2 numeric id
+	 * @return {number}
+	 */
+	getNumericId(): number {
+		// tslint:disable-next-line:radix
+		return parseInt('0x' + this.id.split('-')[4])
```
[the remark this answers is not in the document]
Comment (hubjac1) [reply packages/ozone-helper/ozone-media-url/src/ozone-media-url.ts]: radix 16 will avoid need of tslint disable. But I don't get the point with 'string concatenation'. Cloud you explain?

**Pull request merged**